### DISCUSSION TITLE
Installed MBP Intel i9 19in to use with Apple 27in LED -- My fault

### DISCUSSION THREAD

**Discussion by cowboytek** [MonitorControl/MonitorControl#1522] [Q&A] [answered]
I should have left it alone, but I installed MonitorControl and moved the brightness slider for the Apple 27 inch LED display; not to solve a problem, but just simply to see how it would work should I want to dim the brightness on the display. And then Boom! My LED monitor brightness is stuck at estimated 70% brightness. I have tried all that I could think of to revert the LED back to full brightness with no luck. Troubleshooting: a) toggled various settings in MC b) un/replugged LED cable c) restarted MBP, d) un/replugged LED power, d) AppCleaner MC uninstall e) installed BetterDisplay (same behavior) f) uninstalled BD g) installed MC Lite from AppStore... Please tell me I didn't permanently damage my LED monitor. Any help appreciated.

**Comment by waydabber** [comment, marked as answer]
Hi there - BetterDisplay does not use DDC for obsolete (with no current macOS control support) Apple displays by default. You can set the display to be treaded as a third party display, then DDC control should work with most of these displays. You can change the DDC range under `DDC Features...`:

<img width="700" alt="Screenshot 2024-02-13 at 16 26 39" src="https://github.com/MonitorControl/MonitorControl/assets/37590873/47f1626f-7685-44dc-900b-175098782964">

You need to set the DDC control range to 0..255 instead of 0..100 to access/restore the full scale. MonitorControl Lite does not use DDC either so it can't help. None of the troubleshooting methods you listed do not restore the full brightness scale. You can set the DDC range in MonitorControl as well.

**Comment by cowboytek** [comment]
Wow! Thank you for your really quick response and resolution. That fixed it! Display brightness restored! Made a contribution as a tangible appreciation. Cheers
